subprojects/soak/src/integTest/groovy/org/gradle/resolve/DependencyResolutionStressTest.groovy

```diff
@@ -68,7 +66,9 @@ configurations {
 }
 
 dependencies {
-    compile 'org.gradle:changing:1.0'
+    compile('org.gradle:changing:1.0') {
+        changing = true
+    }
```

Comment:
I don't think this will have any impact on the test:
- We aren't setting the cache timeout (`cacheChangingModulesFor`), so unless the test runs over 24hrs, the module will be taken from the cache anyway.
- We are running the build with `--refresh-dependencies`, to force a cache refresh on every run.